dojo/finding/views.py

```diff
@@ -2147,6 +2154,7 @@ def reset_finding_duplicate_status_internal(user, duplicate_id):
     if duplicate.duplicate_finding:
         # duplicate.duplicate_finding.original_finding.remove(duplicate)  # shouldn't be needed
         duplicate.duplicate_finding = None
+        duplicate.duplicate_finding_id = None
```

Comment:
not needed

Reply:
Still kinda dirty when it pops up in ptvsd mode.